doc/SPONSORS.in

```diff
@@ -7,16 +7,11 @@ The Squid Software Foundation - http://foundation.squid-cache.org/
 	providing the infrastructure and support framework for Squid
 	developers and users.
 
-@Squid-6:
-DigitalOcean - https://www.digitalocean.com/
-
-	DigitalOcean has donated droplets from their cloud infrastructure
-	to host most of Squid Project's continuous integration farm.
+@Squid-7:
+GitHub - https://github.com/
```

Comment:
I do not think SPONSORS should contain organizations that provide free services that simply happened to be used by the Project. IMO, to deserve a mention in this file, the organization has to do something specific to the Squid Project (e.g., give a donation to the Project or give a discount to the Project that we would not receive without a Squid-specific action by that organization).

There is an arguably gray area of organizations that provide free services to _open source_ projects. However, GitHub is not in that gray area -- they provide free service to all kinds of projects, including open and closed source.

Reply:
They have provided a large amount of free bandwidth and recently CI resources to the project. I see no good reason to exclude them on pedantic terminology definitions when the same privilege has already been granted to all previous repository hosts without such consideration.

Reply:
At least some of the "previous hosts" provided Squid-specific contracts/grants/exceptions/etc. At any rate, I disagree that the principle sketched in my change request is based on "pedantic terminology definitions", and I see no good reason to extend past mistakes (if any) to new entries.